### PR TITLE
[TRANSLATION] Update Traditional Chinese translation.

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

**Comment by katahiromz** [comment]
Please fix build.

**Comment by njlyf2011** [comment]
Sorry,I don't understand what your mean.

**Comment by njlyf2011** [comment]
I found out where there was a problem.
There is a problem with the translation file in Traditional Chinese, which causes the compilation to fail.
What should I do? Remove zh-TW.rc from syssetup.rc? 
However, this will result in the inability to appear in Traditional Chinese translation of ReactOS Setup.

**Comment by njlyf2011** [comment]
I am going to delete zh-TW.rc and then re-translate from en_US.rc.

**Comment by katahiromz** [comment]
Text encoding should be UTF-8 without BOM.

**Comment by njlyf2011** [comment]
I may not use UTF-8 BOM. The default setting for my editor is UTF-8 instead of UTF-8 BOM.

**Comment by njlyf2011** [comment]
According to the current compilation results, there is no problem with zh-TW, but there is a problem with zh-CN, I am looking for the reason.

**Comment by katahiromz** [comment]
You need #endif.

**Comment by njlyf2011** [comment]
Sorry.
I have fixed.

File: dll/win32/syssetup/lang/zh-TW.rc [hunk 1]
```diff
@@ -147,38 +167,39 @@ BEGIN
...
-    IDS_GETPROCADDR_FAILED "GetProcAddr 失敗: "
-    IDS_REGSVR_FAILED "DllRegisterServer 失敗: "
-    IDS_DLLINSTALL_FAILED "DllInstall 失敗: "
-    IDS_TIMEOUT "在註冊期間超時"
+    IDS_REGISTERING_COMPONENTS "正在註冊元件..."
+    IDS_LOADLIBRARY_FAILED "LoadLibrary failed: "
+    IDS_GETPROCADDR_FAILED "GetProcAddr failed: "
+    IDS_REGSVR_FAILED "DllRegisterServer failed: "
+    IDS_DLLINSTALL_FAILED "DllInstall failed: "
+    IDS_TIMEOUT "Timeout during registration"
```
[the remark this answers is not in the document]
Comment (njlyf2011) [reply dll/win32/syssetup/lang/zh-TW.rc]: > Why untranslated?

Since it was re-translated from the source file, some entries were missing. I'm Sorry.

**Comment by njlyf2011** [comment]
When will my PR be merged?

**Pull request merged**